bedrock/firefox/templates/firefox/firstrun/firstrun-quantum.html

```diff
@@ -38,10 +59,13 @@ <h1>
           {{_('Sign in to your account and we’ll sync the bookmarks, passwords and other great things you’ve saved to Firefox on other devices.')}}
         {% endif %}
         </p>
+        <p><a href="{{ url('firefox.accounts') }}">{{_('Learn more about Firefox Accounts')}}</a></p>
       {% endblock %}
       </header>
       <div id="fxaccounts-wrapper">
-        <a class="button" href="{{ url('firefox.accounts') }}">{{_('Learn more about Firefox Accounts')}}</a>
+        {{ fxa_email_form(entrypoint='firstrun-page', utm_source='firstrun-page', utm_params={'campaign': 'fxa-embedded-form', 'content': 'firstrun-page', 'medium': 'referral'}, button_class='mzp-c-button mzp-t-product',) }}
+
+        <p class="fxa-signin">{{ _('Already have an account?') }} <a {{ fxa_link_fragment(entrypoint='accounts-page', utm_params={'campaign': 'fxa-embedded-form', 'content': '/firefox/firstrun/', 'source': 'firstrun-page'}) }}>{{ _('Sign In') }}</a></p>
```

Comment:
Nit: suggest chanign to `'content': 'firefox-firstrun'`? Looking at other pages, we seem to favor dashes over trailing slashes?